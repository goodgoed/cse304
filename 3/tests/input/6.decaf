class Overloading {
  void print(int x) {
    // Implementation
  }

  void print(float x) {
    // Implementation
  }
}