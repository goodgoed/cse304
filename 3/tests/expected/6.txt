--------------------------------------------------------------------------
Class Name: Overloading
Superclass Name: 
Fields:
Constructors:
Methods:
METHOD: 1, print, Overloading, private, instance
Method Parameters: 1
Variable Table:
VARIABLE: 1, x, formal, int
Method Body:
Block([
])
METHOD: 2, print, Overloading, private, instance
Method Parameters: 2
Variable Table:
VARIABLE: 2, x, formal, float
Method Body:
Block([
])
--------------------------------------------------------------------------
